pkg/worker/client.go

```diff
@@ -85,14 +85,15 @@ func (c ScrapeAndDumpClient) scrapeAndDump(ctx context.Context) error {
 	for _, sd := range c.workList.GetWork() {
 		reader, err := sd.Scraper.FindSchedules(ctx)
 		if err != nil {
-			return err
+			c.logger.Error(err.Error())
+			continue
 		}
 		defer reader.Close()
 		t := time.Now().UTC()
 		path := fmt.Sprintf("%s/%s.json", sd.Scraper.Prefix(), t.Format(time.RFC3339))
 		err = sd.Dumper.Dump(ctx, reader, path)
 		if err != nil {
-			return err
+			c.logger.Error(err.Error())
 		}
 	}
 	return nil
```

Comment:
This doesn't need a return anymore.

Reply:
Hm, I think the github UI just seems weird here -- the `return err` was removed!

Reply:
Ahh, I see what you're saying now!